components/wifi_httpdhandler/src/wifihandler/handler.cpp

```diff
@@ -0,0 +1,102 @@
+#include "handler.hpp"
+
+#include <cstring>
+#include <unordered_map>
+#include "tcpip_adapter.h"
+#include "esp_err.h"
+#include "esp_log.h"
+
+#include "httpd/util/uri.hpp"
+
+namespace httpd
+{
+  namespace handlers
+  {
+    namespace wifi
```

Comment:
If the component is named `wifihandler` (or should that be `wifi_handler`?), why is the namespace `httpd::handlers::wifi`, that does not match.